src/app/dataModels/project.ts

```diff
@@ -8,4 +8,13 @@ export interface Project {
   languages: string[];
   technologies?: string[];
   programAreas?: string;
+  location?: string;
+  status?: ProjectStatusTypes;
+}
+
+export enum ProjectStatusTypes {
```

Comment:
What is this for?

Reply:
![image](https://github.com/OpenBCca/openbc-web/assets/19850401/92bdd809-cfa0-49a6-989a-cdf99f55bd71)

I have just added the fields present in the project components to the type. ProjectStatusTypes encompass various statuses a project can hold, such as Active, Completed, Done, etc. As for Location, it could typically be "remote" or a specific city, although it might consistently remain remote.

Reply:
Ok,

Can you pass this onto @nam-m and remove it from your PR.

We want to have a clean history of what and where objects are created from.

Reply:
rolled the datatype back